deploy/examples/pool-mirrored.yaml

```diff
@@ -6,7 +6,7 @@
 apiVersion: ceph.rook.io/v1
 kind: CephBlockPool
 metadata:
-  name: mirrored-pool
+  name: mirroredpool
```

Comment:
Instead of changing this example to remove the dash, could you update the doc to add the dash? If someone is using the examples, this way it will be less disruptive.

Reply:
yeah sure! updated, thanks!